adapters/oidc/adapter-core/src/main/java/org/keycloak/adapters/KeycloakDeploymentBuilder.java

```diff
@@ -181,6 +183,8 @@ public HttpClient call() {
     }
 
     public static KeycloakDeployment build(InputStream is) {
+        CryptoIntegration.setClassLoader(KeycloakDeploymentBuilder.class.getClassLoader());
+        log.debugv("BouncyCastle provider: {0}", BouncyIntegration.PROVIDER);
```

Comment:
Maybe CryptoIntegration should log this instead?

Reply:
+1, will update that